docs/scraping-service.md

```diff
@@ -151,3 +149,16 @@ Note that there are no instance configs present in this example; instance
 configs must be passed to the API for the Agent to start scraping metrics.
 See [the docker-compose Scraping Service Example](../example/README.md)
 for how to run a Scraping Service Agent cluster locally.
+
+## agentctl
+
+`agentctl` is a tool included with this repository that helps users to interact
+with the new Config Management API. Currently, it has one subcommand,
```

Comment:
nit: Remove `Currently, it has one subcommand,` and replace it with phrasing that's more future proof ex: "The `agent config-sync` command uses..."